doc/source/reference/changelog.rst

```diff
@@ -191,9 +191,9 @@ Minor Enhancements and Bugfixes
   fixed by `PR 2200 <https://github.com/yt-project/yt/pull/2200>`__.
 - fix an issue related to creating a ds9 region with 
   FITS `PR 2335 <https://github.com/yt-project/yt/pull/2335>`__.
-- add a check to see if pluginfilename is specified in 
+- add a check to see if ``plugin_filename`` is specified in
```

Comment:
Just realised I'm not confortable with retroactively changing the name of the parameter in release notes
```suggestion
- add a check to see if ``pluginfilename`` is specified in
```